### ISSUE TITLE
Panel should be hidden from screen readers during transition animations

### ISSUE THREAD

**Issue by dbjorge** [OfficeDev/office-ui-fabric-react#7758]
When a `Panel`'s `isOpen` prop is set to false, the Panel currently remains visible to screen readers as it goes through its close animation, which results in spurious notifications when child elements of the panel change to support the panel closing (eg, disabling buttons during the close animation) in ways that a screen reader would normally read.

The motivating example in our particular repro is "when a keyboard user presses a button that closes the panel it is contained in, and we disable the button during the panel close animation, a screen reader reads a spurious 'Unavailable' message as the button briefly becomes disabled but is still present and focused in the DOM".

### Environment Information 
 
- __Package version(s)__: 6.127.0 
- __Browser and OS versions__: Chrome Stable (Version 71.0.3578.98 (Official Build) (64-bit)) / Win10

### Please provide a reproduction of the bug in a codepen:
 
1) Open https://codepen.io/dbjorge/pen/BMyLqW
2) Open NVDA
3) Use keyboard tabbing to navigate to the "Close Panel" button
4) Hit enter

#### Actual behavior:
 
NVDA reads out a spurious "unavailable" as the button in the panel becomes disabled during the close animation

#### Expected behavior:
 
The Panel should apply an `aria-hidden="true"` attribute to the panel during the `!isOpen && isAnimating` state to prevent screen readers from reading out any DOM changes that occur during the close animation.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? **No**
Requested priority: **Normal**
Products/sites affected: **N/A** (found in non-public product)

**Comment by dbjorge** [comment]
https://codepen.io/dbjorge/pen/gqpwep has an example that applies an `aria-hidden={!this.state.isPanelOpen}` prop explicitly to the panel, which we've confirmed appears to have the desired behavior of suppressing the spurious notifications in both jaws and nvda.

**Comment by leddie24** [comment]
Thanks @dbjorge for finding this and your suggestion for a fix as well.  I'll submit a PR for this soon!
